wren-ai-service/eval/preparation.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+from pathlib import Path
+
+import orjson
+
+
+def gen_eval_preparation_data_from_json_to_csv(mdl_path: str):
+    assert Path(mdl_path).exists(), f"File not found: {mdl_path}"
```

Comment:
assertions can be disabled with the -O option or by setting the PYTHONOPTIMIZE environment variable to a non-empty string. If we expect this behavior to extend beyond debugging purposes, using a conditional statement might be a better approach.

See also:
- https://docs.python.org/3/using/cmdline.html#cmdoption-O
- https://docs.python.org/3/using/cmdline.html#envvar-PYTHONOPTIMIZE

Reply:
I'll keep it as of now